sites/rcvfortexas-1/contact_elected_official.html

```diff
@@ -0,0 +1,184 @@
+  }
+
+  function displayOfficial(offices, officials, officialTitle, $div) {
+    var officialIndices = getOfficialIndices(officialTitle, offices);
+    $div.empty();
+    officialIndices.forEach(function(officialIndex) {
+      var official = officials[officialIndex];
+      if (official.urls && official.urls.length > 0) {
+        var $name = $('<div><a>' + official.name + '</a><div>');
+        $name.children().attr('href', official.urls[0]);
+      } else {
+        var $name = $('<div>' + official.name + '<div>');
+      }
+      $div.append($name);
+      if (official.emails && official.emails.length > 0) {
+        var $email = $('<div><a>' + official.emails[0] + '</a><div>');
+        $email.children().attr('href', 'mailto:' + official.emails[0]);
+        $div.append($email);
+      }
+      if (official.phones && official.phones.length > 0) {
+        var $phone = $('<div><a>' + official.phones[0] + '</a><div>');
+        $phone.children().attr('href', 'tel:' + official.phones[0]);
+        $div.append($phone);
+      }
+    })
+  }
+
+  /**
+   * Displays what google guesses is the user's address based on user input.
+   */
+  function showAddress(normalizedInput) {
+    var $addressDisplay = $('#resultsAddress');
+    $addressDisplay.empty();
+    var cityState = normalizedInput.city + ', ' + normalizedInput.state;
+    $addressDisplay.append('<div>' + normalizedInput.line1 + '</div>');
+    $addressDisplay.append('<div>' + cityState + '</div>');
+  }
+
+  function showRepresentatives(response) {
+    showAddress(response.normalizedInput);
+    displayOfficial(response.offices, response.officials, 'TX State Senator', $('#txSenate'));
+    displayOfficial(response.offices, response.officials, 'TX State Representative', $('#txHouse'));
+    displayOfficial(response.offices, response.officials, 'U.S. Senator', $('#usSenate'));
+    displayOfficial(response.offices, response.officials, 'U.S. Representative', $('#usHouse'));
+    $('#results').show();
+  }
+
+  function resetForm() {
+    var $button = $('#getRepresentative');
+    $button.prop('disabled', false);
+    var $input = $('#address');
+    $input.val('');
+    $input.attr('placeholder', 'Update Texas Address');
+  }
+
+  function handleResponse(data) {
+    if (data.normalizedInput.state === 'TX') {
+      showRepresentatives(data);
+    } else {
+      $('#errorWrongState').show();
+    }
+    resetForm();
+  }
+
+  function handleFailure(data) {
+    console.log(data.responseJSON);
+    if (data.responseJSON.error.code === 400) {
+      $('#errorUnknownAddress').show();
+    } else {
+      $('#errorUnknown').show();
+    }
+    resetForm();
+  }
+
+  function getRepresentativeInfoByAddress(address) {
+    $('#getRepresentative').prop('disabled',true);
+    $('.errors').hide();
+    // This api key has a restricted set of HTTP referrers.
+    // Check with Irene Foelschow if you need it updated.
+    var apiKey = 'AIzaSyAaRwLVjwQzJZPOxR-FplEAmQGRlzqImB0';
```

Comment:
This has been updated to an RCV for Texas API key on my local file and for the preview. I'll wait to update here until I get more feedback.

Reply:
I don't understand what feedback you need here. The key should be for RCV so I guess I can't merge the PR until it's updated. 

I checked out the page and it works well. I did notice a missing 't' in the word 'representatives' (in the button).

Reply:
Looks good to me.